share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
Potentially worth noting is that some stuff does perform better if you compile from source locally, particularly crypto related things. There are also some things we don't bottle features of because can't absolutely guarantee they'll be supported identically across various CPUs, such as `aesni`.

I think there's a legitimate use case for from source builds, but I'm fine with us scrapping the "Here's an option to always build from source unconditionally" notation in the manpage at least.